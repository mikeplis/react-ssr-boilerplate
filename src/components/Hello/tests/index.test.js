/**
 * You could have a single storyshots.test.js file at the root of the project and call initStoryshots once
 * to initialize all of your storyshot tests, but then all of the generated snapshots would be in that single folder.
 * Also, initializing storyshot tests for each component gives you the flexibility to configure them differently
 * for different components (e.g. if one uses shallow rendering but another doesn't)
 */
import initStoryshots, { shallowSnapshot } from '@storybook/addon-storyshots';

// initStoryshots({
//     test: shallowSnapshot,
//     storyKindRegex: /^Hello$/
// });
initStoryshots({
    storyKindRegex: /^Hello$/
});
