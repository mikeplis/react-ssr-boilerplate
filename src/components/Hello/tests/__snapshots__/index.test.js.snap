// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hello /> renders data 1`] = `
<div>
  <Unknown />
  <h1>
    Hello World!
  </h1>
</div>
`;

exports[`<Hello /> renders loading 1`] = `
<div>
  <Unknown />
  <div>
    Loading...
  </div>
</div>
`;
